paper.md

```diff
@@ -1,32 +1,150 @@
 ---
-title: 'sourmash: a library for MinHash sketching of DNA'
+title: 'sourmash: a tool to quickly search, compare, and analyze genomic and metagenomic data sets'
 tags:
+  - FracMinHash
   - MinHash
   - k-mers
   - Python
+  - Rust
 authors:
- - name: C. Titus Brown
-   orcid: 0000-0001-6001-2677
-   affiliation: University of California, Davis
  - name: Luiz Irber
    orcid: 0000-0003-4371-9659
    affiliation: University of California, Davis
-date: 13 Sep 2016
+ - name: N. Tessa Pierce-Ward
+   orcid: 0000-0002-2942-5331
+   affiliation: University of California, Davis
+ - name: Mohamed Abuelanin
+   orcid: 0000-0002-3419-4785
+   affiliation: University of California, Davis
+ - name: Harriet Alexander
+   orcid: 0000-0003-1308-8008
+   affiliation: Woods Hole Oceanographic Institute
+ - name: Abhishek Anant
+   orcid:  0000-0002-5751-2010
+   #affiliation: @@@
+ - name: Keya Barve
+   orcid: 0000-0003-3241-2117
+   affiliation: University of California, Davis
+ - name: Colton Baumler
+   orcid: 0000-0002-5926-7792
+   affiliation: University of California, Davis
+ - name: Olga Botvinnik
+   orcid: 0000-0003-4412-7970
+   affiliation: Bridge Bio
+ - name: Phillip Brooks
+   orcid: 0000-0003-3987-244X
+   affiliation: Ancestry
+ - name: Peter Cock
+   orcid: 0000-0001-9513-9993
+   #affiliation: @@@
```

Comment:
Hi @peterjc, I am updating the list of sourmash authors to release a JOSS paper for version 4.x. Can you please reply with your desired affiliation? Thanks!

(Of course, also please let me know if you'd prefer not to be included as an author.)

Reply:
Looking back my only contribution to date was a tiny cosmetic documentation fix, so I'll decline - but thank you for asking.

Reply:
got it. Thank you for the response and the contribution!